active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -0,0 +1,894 @@
+
+AnimatedIcon
+===
+
+# Background
+
+XAML has a couple of base types for showing an icon, aside from simply showing a small image:
+* [IconElement](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconElement)
+* [IconSource](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconSource)
+
+IconElement & IconSource have the same subclasses for different kinds of icons, for example
+[BitmapIcon](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIcon)
+and [BitmapIconSource](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIconSource).
+This spec as adding an `AnimatedIcon` and `AnimatedIconSource`.
+
+The duplication of two icon types is cause for confusion,
+but trying to solve that underlying issue is outside the scope of this spec.
+The ultimate difference between the two is that some controls require an IconElement and some
+require an IconSource. The new animated icon APIs here are continuing to use the existing pattern.
+
+AnimatedIcon (and AnimatedIconSource) displays an animation defined by a Lottie animation,
+similar to the existing
+[AnimatedVisualPlayer](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer)
+element.
+A Lottie animation is typically defined in Adobe After Effects and exported to JSON, then the JSON
+is converted to a WinRT class using the LottieGen tool.
+The new APIs in this spec include classes created by LottieGen.
+
+_Spec note: Rather than using LottieGen to create new public APIs, we could ship JSON files
+and then parse those, and there is a toolkit control that does that. But the perf
+is better to code gen, and the toolkit control requires C#._
+
+# AnimatedIcon How-To
+
+_(This is conceptual documentation that will go to a docs.microsoft.com "how to" page)_
+
+An AnimatedIcon can be used to display an animation that's defined in Adobe After Effects,
+such as the PlayAnimation in this example:
+
+```xml
+<AnimatedIcon>
+    <local:PlayAnimation />
+</AnimatedIcon>
+```
+
+This PlayAnimation can similarly be used in an 
+[AnimatedVisualPlayer](http://msdn.microsoft.com/library/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer):
+
+```xml
+<AnimatedVisualPlayer>
+    <local:PlayAnimation />
+</AnimatedVisualPlayer>
+```
+
+The difference is that AnimatedVisualPlayer has methods such as Play and Pause.
+AnimatedIcon controls the animation with a State property, and can color the animation using the Foreground property.
+AnimatedIcon can also be used where specfically an Icon-typed value is required.
```

Comment:
Another differences between the new AnimatedIcon and existing AnimatedVisualPlayer is that they consume different IAnimatedVisualSource interfaces.